man/systemd.service.xml

```diff
@@ -798,11 +798,14 @@
         notification socket, as accessible via the
         <citerefentry><refentrytitle>sd_notify</refentrytitle><manvolnum>3</manvolnum></citerefentry>
         call. Takes one of <option>none</option> (the default),
-        <option>main</option> or <option>all</option>. If
-        <option>none</option>, no daemon status updates are accepted
-        from the service processes, all status update messages are
-        ignored. If <option>main</option>, only service updates sent
-        from the main process of the service are accepted. If
+        <option>main</option>, <option>exec</option> or
+        <option>all</option>. If <option>none</option>, no daemon status
+        updates are accepted from the service processes, all status
+        update messages are ignored. If <option>main</option>, only
+        service updates sent from the main process of the service are
```

Comment:
Hm, we should fix the `main`-description too. Actually, `root` can send "fake"-ucred data, i.e.
```
User=0
NotifyAccess=main
ExecStart=/bin/sh -x -c 'sleep 2; systemd-notify --ready; sleep 10'
``` 
works fine

But this unrelated to this PR, of course